src/SSW.Rewards.Application/Common/Behaviours/UnhandledExceptionBehaviour.cs

```diff
@@ -11,7 +11,7 @@ public UnhandledExceptionBehaviour(ILogger<TRequest> logger)
         _logger = logger;
     }
 
-    public async Task<TResponse> Handle(TRequest request, CancellationToken cancellationToken, RequestHandlerDelegate<TResponse> next)
+    public async Task<TResponse> Handle(TRequest request, RequestHandlerDelegate<TResponse> next, CancellationToken cancellationToken)
```

Comment:
can you pass the cancellation token down into `next()` as well?

`await next(cancellationToken);`

Reply:
A per above comment, can check that in https://github.com/SSWConsulting/SSW.Rewards.API/issues/65

Reply:
No:
```csharp
/// <summary>
/// Represents an async continuation for the next task to execute in the pipeline
/// </summary>
/// <typeparam name="TResponse">Response type</typeparam>
/// <returns>Awaitable task returning a <typeparamref name="TResponse"/></returns>
public delegate Task<TResponse> RequestHandlerDelegate<TResponse>();
```

I think the cancellation token here is in case you perform any async logic in your behavior.